documentation/documentation.html

```diff
@@ -622,7 +622,125 @@ <h2 class="main-h2">
 
             </div>
 
+            <div class="badge">
+                <div class="comp-intro">
+                    <h1 class="main-h1">Badges</h1>
+                    <p class="main-p weight-mid">Documentation and examples for badges, our small count and labeling
+                        component.</p>
+                </div>
+                <h2 class="main-h2">Example of Avatars with badges</h2>
+                <p class="doc-p">
+                    Avatar Badges are available in 2 types- online badge, offline bagde.
+                    <br>
+                    To use these badges use classes <span class="prime-color-text">badge-container</span> along with
+                    classes
+                    to style the badge like
+                    <span class="prime-color-text">online-img-badge, offline-img-badge</span>.
+                    <br>
+                    <br>
+                    <code class="main-code">
+                        badge-container:
+                    </code>
+                    For the parent div
+                    <br>
+                    <code class="main-code">
+                        online-img-badge:
+                    </code>
+                    For green online badge.
+                    <br>
+                    <code class="main-code">
+                        offline-img-badge:
+                    </code>
+                    For red offline img badge.
+                    <br>
+                    <br>
+                    You can copy html part from below code snippet.
+
+                </p>
+                <div class="comp-container">
+                    <div class="badge-container">
+                        <img src="../components/avatars/images/avatar.png" class="avatar-circle avatar-big">
+                        <div class="online-img-badge"></div>
+                    </div>
+                    <div class="badge-container">
+                        <img src="../components/avatars/images/avatar.png" class="avatar-circle avatar-mid">
+                        <div class="online-img-badge"></div>
+                    </div>
+                    <div class="badge-container">
+                        <img src="../components/avatars/images/avatar.png" class="avatar-circle avatar-small">
+                        <div class="online-img-badge"></div>
```

Comment:
I think you can use **span** instead of **div** tags.

Other than that everything is good.

Reply:
Yes used the div tag as users generally use div tags and not span. So for convenience!